Services/Documents/DocumentService.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Andculture.Sitefinity.Testing.Models.Configuration;
+using Andculture.Sitefinity.Testing.Models.Content.Documents;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using RestSharp;
+using System.IO;
+
+namespace Andculture.Sitefinity.Testing.Services.Documents
+{
+    public class DocumentService : ODataService<Document>
+    {
+        public DocumentService(TestingSettings testingSettings, string accessToken)
+            : base(testingSettings, accessToken)
+        {
+            EndpointUrl = "/sf/system/documents";
```

Comment:
Suggestion - Consider moving the path to a configuration store. That way we can quickly update it if it changes over time.